README.md

```diff
@@ -1,11 +1,11 @@
-<img src="./logo.png" alt="Terra Draw Logo" width="400"/>
+<img src="./readme.gif" alt="Terra Draw Logo" width="400"/>
```

Comment:
We don't want to change the logo here I don't think. Can we undo this change?

Reply:
Sure